sdk/identity/identity/src/tokenCache/InMemoryPersistence.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as msalCommon from "@azure/msal-common";
+import { CachePlugin } from "./types";
+
+/**
+ * IMPORTANT:
+ * Do not use this outside the Identity source.
+ * Do not instantiate this more than once per credential instance.
```

Comment:
what if someone pulls in different versions of this package? Do we want to try using global variables/symbols to share across package versions?

Reply:
🤔 I'm not sure I got it! This persistence will only be created once per credential instance, and it will be inaccessible to the users. We also want o remove this from our codebase asap. We're waiting for MSAL to help us resolve this.